tests/test_signing.py

```diff
@@ -168,38 +201,6 @@ def test_invalid_signed_message(self):
             forged = SignedMessage(signature + message)
             skey.verify_key.verify(forged)
 
-    def test_base64_smessage_with_detached_sig_matches_with_attached_sig(self):
```

Comment:
We shouldn't remove these tests until we're ready to remove the encoder entirely. Just `with pytest.raises` on the encoder lines.

Reply:
Both these test where added in #504 the lines are deleted from the `Revert "Decode smessage" #504` commit.